cypress-tests/cypress/support/commands.js

```diff
@@ -775,17 +712,12 @@ Cypress.Commands.add("listRefundCallTest", (globalState) => {
       "Content-Type": "application/json",
       "api-key": globalState.get("apiKey"),
     },
-    body:{"offset":0}
+    body: { "offset": 0 }
```

Comment:
this is not exactly required, if body is not need for this request, u can remove the field

Reply:
This is required. If no body is passed, we get `content type error` as shown below:
```json
{
	"error": "Content type error"
}
```

Reply:
got it, then move this request body to a separate json file